source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -34,17 +34,16 @@
     <PackageReference Include="Assent" Version="1.6.1" />
     <PackageReference Include="Polly" Version="8.3.1" />
     <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <ProjectReference Include="..\Calamari.Aws\Calamari.Aws.csproj" />
+    <ProjectReference Include="..\Calamari.Azure\Calamari.Azure.csproj" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
     <PackageReference Include="Markdown" Version="2.1.0" />
+    <PackageReference Include="NGitLab" Version="10.4.1" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.3.0" />
-    <ProjectReference Include="..\Calamari.Aws\Calamari.Aws.csproj" />
```

Comment:
These two project references are the same in each framework. Minor drive by cleanup